setup.py

```diff
@@ -80,7 +80,7 @@ def build_description():
     ],
 
     install_requires=[
-        "PyYAML >= 3.10, < 4",
+        "PyYAML >= 3.10, < 6",
```

Comment:
Thanks! Let's require at least 5.1 here.

Reply:
OK, if we require at least 5.1 (added in 
9b8ba29), we could also use `FullLoader` (added in 
05eba75).